tests/functional/validate/08-whitespace/flow.cylc

```diff
@@ -5,10 +5,6 @@
     initial cycle point = 20140101T00
     [[  special tasks  ]]    
         sequential = foo , bar ,  baz   
-        include at start-up =   \
-            foo {% if HELLO %}   ,    \
-            bar    
-            {% endif %}   
```

Comment:
Could this be moved to the value of a different setting, so we can still test the whitespace of this arrangement?

Reply:
Can you explain exactly what this is trying to test. I can't really understand what this is for.

Reply:
Just to note what we discussed today: we think this tests horribly formatted lists, but seeing as lists are dying off in the Cylc8 config, let's just get rid of this test